### ISSUE TITLE
Offline status

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-call#926]
#### What would you like to do?

EC doesn't warn you if you've lost your connectivity.

#### Why would you like to do it?

To avoid sitting there like a lemon wondering why the other user hasn't joined.

#### How would you like to achieve it?

Some pretty UI to tell you how connected you are to the internet or not.  Could also be an extension of warning you about your own network quality in general.

**Comment by rajil** [comment]
It is now part of the spec [here](https://github.com/matrix-org/matrix-spec-proposals/blob/toger5/matrixrtc-call-ringing/proposals/4075-rtc-notification-event.md#ringing-acknowledgements). Is there any plan to implement this soon?

**Comment by Johennes** [comment]
@rajil what you linked to helps communicate the callee's offline state while ringing them. This issue is about _your own_ connectivity.

**Comment by rajil** [comment]
@Johennes I agree but @SimonBrandner shot down my issue [here](https://github.com/element-hq/element-call/issues/922)
